packages/core-components/src/components/checkbox-group/checkbox-group.tsx

```diff
@@ -74,8 +78,8 @@ export class CheckboxGroupComponent {
         node.disabled = true;
       });
     } else {
-      nodes.forEach(node => {
-        node.disabled = false;
+      nodes.forEach((node, index) => {
```

Comment:
Won't it be a problem to rely on index ?

Reply:
For instance,
```
<g>
<c id="1">
<c id="2" disabled>
<g>
```

Disable group
```
<g disable>
<c id="1" disabled>
<c id="2" disabled>
<g>
```

Based on some condition render UI
```
<g disable>
<c id="2" disable>
<c id="1" disabled>
<g>
```

Enable group
```
<g>
<c id="2">
<c id="1" disabled>
<g>
```